README.md

```diff
@@ -47,9 +47,18 @@ includes:
     - vendor/prestashop/phpstan-prestashop/extension.neon
 ```
 
-You need to provide a service that is an instance of `PHPStanForPrestaShop\PHPConfigurationLoader\ConfigurationLoaderInterface` named `@strictTypesForNewClassesRuleConfigurationFileLoader`. It is used by rule `UseStrictTypesForNewClassesRule`.
+### Required settings
```

Comment:
Rework of the README to make it clearer - see https://github.com/PrestaShop/phpstan-prestashop/blob/ec12ca64f920fdd8ad931fa26e0e5049e4fc5e11/README.md